articles/multifactor-authentication/custom.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Configuring Custom Multi-factor Authentication
+url: /multifactor-authentication/custom
+description: Examples for configuring custom MFA implementations.
+topics:
+  - mfa
+  - custom-mfa
+contentType:
+  - how-to
+  - concept
+useCase:
+  - customize-mfa
+---
+# Configuring Custom MFA
+
+You may configure a [rule](/rules) from your [Dashboard > Rules](${manage_url}/#/rules) for custom MFA processes, which allow you to define the conditions that will trigger additional authentication challenges, such as changes in geographic location or logins from unrecognized devices. You can also use the rule to force a particular MFA provider despite the Dashboard toggles, or to force MFA only for particular applications or for particular users.
+
+## The provider setting
+
+The `provider` setting now defaults to `any`. This allows the toggles on the [Dashboard > MFA](${manage_url}/#/mfa) to control which providers are enabled, and the user to select from any of the enabled providers. Setting `provider` to `guardian`, for example, will now restrict end users to only use Guardian for their second factor, regardless of which factors are enabled in the Dashboard. In most cases, it would be recommend to leave the setting as is, rather than modifying it with a rule, and instead control the available factors via the Dashboard toggles.
+
```

Comment:
Maybe something like:

You can also set the 'provider' to any of these legacy options, which are kept just for backwards compatibility:

'guardian' = uses SMS or Push
'google-authenticator' = forces you to use Google Authenticator specific UI
'duo' = forces you to use DUI

Reply:
Guardian is equivalent to `any` at the moment. If you set `provider=guardian` but enable just OTP, you will be allowed to enroll/verify just with OTP. `provider=guardian` (and `provider=any` of course) also lets you use the email authenticator on the MFA API if email is enabled.